phoenix-core/src/test/java/org/apache/phoenix/parse/QueryParserTest.java

```diff
@@ -66,6 +68,46 @@ private void parseQueryThatShouldFail(String sql) throws Exception {
         }
     }
 
+    @Test
```

Comment:
Do the permissions/user/group have to be in single quotes?

Reply:
Yes. permissions/user/group are treated as literals and table name is treated as identifier.